src/applications/yellow-ribbon/containers/SearchForm/index.jsx

```diff
@@ -225,8 +263,9 @@ export class SearchForm extends Component {
 }
 
 const mapStateToProps = state => ({
-  fetching: state.yellowRibbonReducer.fetching,
-  showMobileForm: state.yellowRibbonReducer.showMobileForm,
+  fetching: getYellowRibbonAppState(state).fetching,
+  showMobileForm: getYellowRibbonAppState(state).showMobileForm,
+  showYellowRibbonEnhancements: yellowRibbonEnhancements(state),
```

Comment:
Can we rename this selector? Also I'm not sure if abstracting these dot properties in selectors makes it more complicated or easier to read, what do you think?

Reply:
I don't mind selectors; just using it since it seems to be something a few people are doing in the react/ redux world.
I can rename it so something like you suggested for help of readability.